api/share/tools.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, List, Union
```

Comment:
I require Python 3.9 (or even 3.10) or higher, so you can leave just the `Union`.

Reply:
Ok, I'll fix it. But I'm sure it isn't the most important things we need to do now. During refactoring - yes, but not now.

Reply:
Absolutely, just see my comment above: I would like to merge things into `master` when questions about code are resolved.

Reply:
Ok, I got it